exchange/peermgr_test.go

```diff
@@ -19,7 +20,7 @@ func TestHeyEvtPeerMgr(t *testing.T) {
 	n1 := testutil.NewTestNode(mn, t)
 	n2 := testutil.NewTestNode(mn, t)
 
-	idx, err := NewIndex(n1.Ds)
+	idx, err := NewIndex(n1.Ds, blockstore.NewGCBlockstore(blockstore.NewBlockstore(n1.Ds), blockstore.NewGCLocker()))
```

Comment:
If you update the `TestNode` to use a GCBlockstore, then you can do:
```suggestion
	idx, err := NewIndex(n1.Ds, n1.Bs)
```